For Ember Times No. 132 - EmberConf Ad 🍓🥪 (#501)
ember-learn/ember-blog: source/2020-01-27-the-ember-times-issue-132.md

---

## [Try your first Octane & JAM sandwich at EmberConf 🏎🥪🍓](https://emberconf.com/#/speakers/chris-manson)

> Surely, you remember him as [the Emberista who built a blog site with Ember & NodeJs on stage in less than 15 mins](https://www.youtube.com/watch?v=GHbjdesHhPA), you heard him talk about [his work on an Ember-powered blog engine](https://www.youtube.com/watch?v=OnDUp2AhuNo) or you

Reviewer: An optional suggestion. I thought using fewer syllables would deliver your punchline better.
```suggestion
Surely, you remember him as [the Emberista who built a blog site with Ember & Node in 15 min](https://www.youtube.com/watch?v=GHbjdesHhPA), you heard him talk about [his work on an Ember-powered blog engine](https://www.youtube.com/watch?v=OnDUp2AhuNo) or you
```

> recall his [jammiest introduction to the JAM (JavaScript, APIs and Markup) stack for humans](https://www.youtube.com/watch?v=eQULGb10MbQ) with [46 mentions of the word "JAM" in total](https://twitter.com/real_ate/status/1207683401446313984)!

Reviewer: I think we can switch the order to add more humor:
```suggestion
recall him [say JAM 46 times](https://twitter.com/real_ate/status/1207683401446313984) in his [jammiest introduction to the JAM (JavaScript, APIs and Markup) stack for humans](https://www.youtube.com/watch?v=eQULGb10MbQ)!
```

Author: Thank you, that sounds great!

This year, [Chris Manson (@mansona)](https://github.com/mansona) has even more insights into the modern JAM stack to share with you!

<!--alex disable king-queen-->

> At [this year's EmberConf](https://emberconf.com/) he's going to teach us about [**An Octane-Powered JAM Stack**](https://emberconf.com/#/schedule/day-1_an-octane-powered-jam-stack); how and why the JAM Stack has become so popular and how Ember Octane blends in well with this approach of web development. But that's not all: you can learn hands-on, how to build you first JAM stack site based on Octane and Empress in his EmberConf workshop [**Build and Publish Your Own empress-blog Template**](https://emberconf.com/#/schedule/pre-conf_build-and-publish-your-own-%60empress-blog%60-template).

Reviewer: It's hard to see the diffs, sorry. I removed the word "how" from "how and why" (it seemed redundant), and edited the part "build you first ... in" to "build your first ... at".
```suggestion
At [this year's EmberConf](https://emberconf.com/) he's going to teach us about [**An Octane-Powered JAM Stack**](https://emberconf.com/#/schedule/day-1_an-octane-powered-jam-stack); why the JAM Stack has become so popular and how Ember Octane blends in well with this approach of web development. But that's not all: you can learn hands-on how to build your first JAM Stack site with Octane and Empress at his EmberConf workshop [**Build and Publish Your Own empress-blog Template**](https://emberconf.com/#/schedule/pre-conf_build-and-publish-your-own-%60empress-blog%60-template).
```

> JAM Stack has become a popular phrase in our industry over the last few years with projects like Gatsby and Vuepress gaining popularity at fantastic speeds. As it is a very new tech stack there is still some innovation that can be added to the mix, and Ember Octane has the power to unlock a whole new generation of JAM Stack developers with the Empress project.

Reviewer: I thought "popular idea" might have a more positive meaning than "popular phrase". The latter reminded me of something hyped unnecessarily.
```suggestion
JAM Stack has become a popular idea in our industry over the last few years. Projects like Gatsby and Vuepress are gaining popularity at fantastic speeds. As it is a new tech stack, there is still some innovation that can be added to the mix, and Ember Octane has the power to unlock a whole new generation of JAM Stack developers with the Empress project.
```

Author: Woops, I copy-pasta'd the abstract from the schedule here for reference while writing the section and forgot to delete it again - will edit and thanks for the heads up!

In an exclusive interview with The Ember Times, Chris shares with us, how popular the JAM stack is and why it is so useful for building modern websites in 2020:

> […] My belief is that most Ember developers are already using JAM Stack concepts. It’s not a binary thing, it’s not “you use the JAM stack” or “you don’t”. Instead you can make your apps more JAM and the question of when you want to do it is answered by when you need it. If you need good SEO, or if you need faster start times, you might want to look at some of the JAM technologies.

What will our experience building JAM stack websites with Octane be like? Chris shares from his experience as a JAM stack developer and as a workshop mentor:

<!--alex ignore just-->
> The thing that most people like when using JAM Stack with Ember, is the fact that you can get so much by just installing 1 or 2 addons. If you had to teach somebody everything from the ground all the way up to getting it deployed, it would take much longer and it would be much more difficult. But leveraging the power of Ember, leveraging the power of Ember addons makes people experience quick wins and gets a lot done very quickly.

> Besides from learning about the JAM stack in theory and in practice - what else makes EmberConf your go-to conference in 2020? Chris shares is perspective on what makes the event so worth being part of:

Reviewer: ```suggestion
Besides learning about the theory and practice of JAM stack, what makes EmberConf your go-to conference in 2020? Chris shares his perspective on what makes the event so worth being part of:
```
